dyninstAPI/src/parse-cfg.h

```diff
@@ -323,7 +323,7 @@ class parse_func : public ParseAPI::Function
         }
     };
 
-#if defined(DYNINST_HOST_ARCH_POWER) || defined(DYNINST_HOST_ARCH_AARCH64)
+#if defined(DYNINST_HOST_ARCH_POWER) || defined(DYNINST_HOST_ARCH_AARCH64) || defined(DYNINST_CODEGEN_ARCH_AMDGPU_GFX908)
    bool savesReturnAddr() const { return saves_return_addr_; }
```

Comment:
I believe this function only used in stackwalker, since we removed every stack walker stuff for AMDGPU, we shouldn't need this for AMDGPU.